examples/plot_fmri_data_example.py

```diff
@@ -151,19 +154,26 @@ def preprocess_haxby(subject=2, memory=None):
 
 SVR_permutation_test_inference = False
 if SVR_permutation_test_inference:
-    # We computed the regularization parameter by CV (C = 0.1)
-    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_cv(
-        X, y, n_permutations=50, C=0.1
+    # We computed the regularization parameter from a cross valisation (C = 0.001)
+    estimator = LinearSVR(C=0.001)
+    weight_svr, weight_svr_distribution = permutation_test(
+        X, y, estimator, n_permutations=50
+    )
+    pval_corr_svr_perm_test, one_minus_pval_corr_svr_perm_test = permutation_test_pval(
+        weight_svr, weight_svr_distribution
     )
 
 # Another method is to compute the p-values by permutation test from the
 # Ridge decoder. The solution provided by this method should be very close to
 # the previous one and the computation time is much shorter: around 20 seconds.
-
-estimator = Ridge()
-pval_corr_ridge_perm_test, one_minus_pval_corr_ridge_perm_test = permutation_test(
+# We computed the parameter from a cross valisation (alpha = 0.0215)
+estimator = Ridge(alpha=0.0215)
```

Comment:
Please use a RidgeCV, so that there is no magic alpha parameter hanging around.

Reply:
See comment above.

Reply:
Cross-validation of LinearSVR: 56s
Instantiation of LinearSVR: >1ms 
Permutation test of LinearSVR: 22O s

Permutation test with RidgeCV: 81.6 s
Permutation test with Ridge: 10.7 s

Reply:
Then another possibility would be to call RidgeCV once to learn the hyperparameter and then plug it into the Permutation test. Does this make computation time more reasonable ?

Reply:
RidgeCV alone: 0.25 s
Permutation test with Ridge based on alpha from RidgeCV: 25.2s
However, using Ridge with the alpha from CV generates warnings and doubles the time because there is a difficulty with convergence.

Reply:
The alpha grid could be improved maybe ?
Normally a Ridge estimator is pretty stable and well-behaved.

Reply:
Using alpha grid is quite equivalent to using RidgeCV. 
In this case, the alpha from CV is based on the relation between X and y. In consequence, the shuffling of y breaks this relation. For more stable behaviour, we should use the default value of alpha.

Reply:
Not exactly. RidgeCV uses an arbitrary  grid that may not contains the correct range of values.
When you use Ridge, you can change the regularizer by a factor of 2 or so, this has limited impact. What you need is just to pick the right magnitude order. 
The default value of alpha may not be of the right order of magnitude (?)
Permuting y only has a big impact  on the optimal alpha of the variance fit by the model is large (e.g. 90%) : I guess it is not the case here ?
As you started the example a while ago with an alpha close to 0.02, can you confirm using grid search that this is the value to pick ?

Reply:
OK, then the question becomes: does it make a true difference in terms of results to use Ridge vs RidgeCV ?
If yes, I think that we should rely on RidgeCV.

Reply:
Using Ridge or RidgeCV in this example provides the same result; there are not significant voxels.

Reply:
Ha, ha. Then I'd go with defaults parameters, without CV, and add a comment stating that ideally you want to use a RidgeCV, but this is not done here for the sake of time.

Reply:
done